.gitignore

```diff
@@ -102,3 +102,9 @@ dist
 
 # TernJS port file
 .tern-port
+
+<<<<<<< HEAD
```

Comment:
This is probably a little mistake from git rebasing.

Reply:
Should be fixed with latest commit